artifacts-wg/charter/charter.md

```diff
@@ -0,0 +1,47 @@
+# Purpose
+In the distributed world of cloud-native computing different artifacts and packages are used to transport configuration and code for the many services and capabilities that comprise and support workloads and applications. As an example, today ArtifactHub advertises that it can crawl about 20 such artifact types [1]. These "cloud-native" bundles add to the previous proliferation of package and artifact types for software dependencies such as Maven and npm and for system packages like deb and rpm.
+
+This abundance of package types and formats adds complexity and risk for cloud application developers and architects trying to provision and run cloud-native workloads. Different tools and controllers must be carefully learned and installed to bundle, unbundle, verify and deploy each artifact type. This complexity slows product development by users, impedes innovation and collaboration amongst projects, and increases risks of insecure configuration.
+
+To reduce this complexity and facilitate collaboration and innovation, WG Artifacts will gather stakeholders from many CNCF and open source projects offering packaging, distribution and deployment mechanisms for bundles of configuration and code.
+
+**The WG will:**  
+* **Develop common patterns**  
```

Comment:
Will you really be developing patterns rather than observing and notating what is in use?

Reply:
I'm hoping that my refactor on the goals section clarifies this a bit. Can you PTAL and let me know if you'd like this bit about patterns reworked?

Reply:
@AlexsJones I believe the group intends to influence common patterns directly. First they'll gather up many existing practices, then they'll share practices and patterns across them, and finally they'll influence and standardize these common practices in OCI specifications.